Example/SnapshotTests/BPKFlareViewSnapshotTest.m

```diff
@@ -75,6 +75,18 @@ - (void)testFlareViewSnapshot {
     FBSnapshotVerifyView(parentView, nil);
 }
 
+- (void)testFlareViewSnapshotTop {
```

Comment:
Maybe we should rename `testFlareViewSnapshot`?

Reply:
That's what the default test is called though. Did you mean `testFlareViewTopSnapshot`?

Reply:
Yeah I am saying we should rename that method given the introduction of `testFlareViewTopSnapshot`

Reply:
Oh I get it.

Clearly I need more coffee!